etabotsite/etabotapp/models.py

```diff
@@ -3,7 +3,7 @@
 import logging
 
 logging.getLogger().setLevel(logging.DEBUG)
-
+logging.info('models import started.')
```

Comment:
this type of message should not be set at `info` level. Suggest change it to `debug`